pkg/browsers/console.go

```diff
@@ -116,13 +132,16 @@ func OpenWithFirefoxContainer(urlString string, labels RoleLabels) error {
 		}
 	}
 
-	tabURL := fmt.Sprintf("ext+granted-containers:name=%s&url=%s", labels.MakeExternalFirefoxTitle(), url.QueryEscape(urlString))
+	tabURL := fmt.Sprintf("ext+granted-containers:name=%s&url=%s", labels.Profile+labels.MakeExternalProfileTitle(), url.QueryEscape(urlString))
```

Comment:
@meyerjrr can you update the MakeExternalFirefoxTitle() function to do this so its reusable?

Also I think you should replace this taburl with a call to MakeFirefoxContainerURL() to avoid duplicating